aspnetcore/host-and-deploy/linux-apache.md

```diff
@@ -62,28 +68,6 @@ app.UseForwardedHeaders(new ForwardedHeadersOptions
 app.UseAuthentication();
 ```
 
-::: moniker-end
```

Comment:
Was removing this intentional?

Reply:
Yes, the topic is versioned now for 2.1+, so we don't need version ranges.

Reply:
See Line 5